modules/manually-create-identity-access-management.adoc

```diff
@@ -48,14 +94,31 @@ compute:
 ...
 ----
 <1> This line is added to set the `credentialsMode` parameter to `Manual`.
+endif::cco-multi-mode[]
 
-. To generate the manifests, run the following command from the directory that contains the installation program:
+. Generate the manifests by running the following command from the directory that contains the installation program:
 +
 [source,terminal]
 ----
 $ openshift-install create manifests --dir <installation_directory>
 ----
++
+where:
+
+`<installation_directory>`:: Specifies the directory in which the installation program creates files.
 
+. Copy the generated credential files to the target manifests directory:
++
+[source,terminal]
+----
+$ cp <path_to_ccoctl_output_dir>/manifests/*credentials.yaml ./path/to/installation/dir/manifests/
+----
++
+where:
+
+`<path_to_ccoctl_output_dir>/manifests`:: Specifies the directory created by the `ccoctl alibabacloud create-ram-users` command.
```

Comment:
@kwoodson @mtulio @joelddiaz Am I correct here? You need to [copy the directory](https://github.com/openshift/cloud-credential-operator/pull/412/files#diff-2480a11ca4927139d6eaa9883946b6f4cb38358cd98def8c57dd73e9319dbc9cR329) created by the `ccoctl alibabacloud create-ram-users` command?

Reply:
Since the installer_dir/manifests/ directory already exists (because the installer already put some of its own files in there), it feels a little weird to say "copy the directory". Technically, you need to copy the files from path_to_ccoctl_output_dir/manifests/* to the already existing directory created by the installer in installer_dir/manifests/.

Reply:
Sorry. Right. Copy the files in the directory...
But `<path_to_ccoctl_output_dir>/manifests` was created when the user runs the `ccoctl alibabacloud create-ram-users --name <name> --region=<alibaba-region> --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests --output-dir=<path_to_ccoctl_output_dir>/manifests` command?

Reply:
Yes. That's where the `ccoctl` command places the Secret files that need to be copied over to the installer.

Reply:
@mburke5678 , I used to run this command (`cp`):
```bash
# create RAM Users
./ccoctl \
   alibabacloud \
   create-ram-users \
  --region ${ALIBABA_REGION_ID} \
  --name $(awk '/infrastructureName:/{print $2}' ${INSTALL_DIR}/manifests/cluster-infrastructure-02-config.yml) \
  --credentials-requests-dir ${PWD}/cco-credrequests \
  --output-dir ${PWD}/cco-manifests

# Copy the manifests to install dir
cp -v ${PWD}/cco-manifests/*/*.yaml ${INSTALL_DIR}/manifests/
```

But we can change the `--output-dir ${PWD}/cco-manifests` to `--output-dir ${PWD}/`, to keep more simple the output dir, then we can use the `cp` like this:
```
cp -v ${PWD}/manifests/*.yaml ${INSTALL_DIR}/manifests/
```